tests/dist/python/run_dist_objects.py

```diff
@@ -197,6 +182,20 @@ def sample_neighbors_test(g):
     sample_neighbors_with_args(g, size=2**12, fanout=1)
 
 def test_dist_graph_services(g):
+    # in_degrees and out_degrees does not support heterograph
```

Comment:
the arguments of `in/out_degrees()` are supposed to be global IDs, so heterograph is supported?

Reply:
It's not implement as it is for the DGLGraph (with etypes):
https://github.com/dmlc/dgl/blob/226d1159a94880c8aeab96b20e821f4186eeef64/python/dgl/heterograph.py#L3427

and even though it might work with gobal IDs, it's seemingly not officially supported by the DistGraph API :
https://github.com/dmlc/dgl/blob/226d1159a94880c8aeab96b20e821f4186eeef64/python/dgl/distributed/dist_graph.py#L910